static/js/chart/types.ts

```diff
@@ -170,6 +170,7 @@ export interface HorizontalBarChartOptions extends ChartOptions {
 
 export interface LineChartOptions extends ChartOptions {
   handleDotClick?: (dotData: DotDataPoint) => void;
+  timeScale?: string;
```

Comment:
Use typing `"year" | "month" | "day"` instead of `string`

Reply:
Good call! Done!